media-video/libav/libav-11.9999.ebuild

```diff
@@ -134,8 +135,9 @@ REQUIRED_USE="
 	amr? ( gpl ) aac? ( gpl ) x264? ( gpl ) X? ( gpl ) cdio? ( gpl ) x265? ( gpl )
 	test? ( encode zlib )
 	fontconfig? ( truetype )
+	?? ( gnutls libressl )
```

Comment:
This looks bad. I would assume that (like with libav), if I want libressl *instead of* openssl, I set USE=libressl globally. Now this forces me to locally alter it in order not to have conflict with gnutls.